app/styles/organisms/panel.less

```
@@ -0,0 +1,60 @@
+.oss-panel {
+  border-radius: var(--radius-border-radius-md, 8px);
+  border: 1px solid var(--color-gray-200, #E5E7EB);
+  background: var(--color-default-white, #FFF);
```

```suggestion
  border-radius: var(--radius-border-radius-md);
  border: 1px solid var(--color-border-default);
  background: var(--color-white);
```
You don't need to set default value when you are using `var`, also `--color-default-white` variable doesn't exist